### ISSUE TITLE
scenic route campaign: Iranian airfields seem to be overpopulated

### ISSUE THREAD

**Issue by niehorst** [dcs-liberation/dcs_liberation#1635]
I face the problem that the opposing force is not able to get jets airborne after mission start when the first flights took off.
The few iranian airfields in this campaign seem to get overpopulated really fast. All aircrafts start engines but no one will taxi to the runway. Can't see anything that is blocking taxiways or something. I didn't change starting budget or income, perhaps the enemy got to much money and there are to less airfields.

**Comment by Starfire13** [comment]
Yeah, DCS AI has trouble with small airfields that don't have a taxiway.
